### ISSUE TITLE
Placing Picker next to an Icon in the same row disappears on Android

### ISSUE THREAD

**Issue by ilaif** [GeekyAnts/NativeBase#457]
Hey guys, Thanks for this repo!
## react-native, react and native-base version
```json
{"react": "~15.4.0-rc.4", "react-native": "0.40.0", "native-base": "^0.5.22"}
```
## Expected behaviour
When using a `<View>` to wrap `<Icon>` and `<Picker>` in the same row, the <Picker> should be visible and touchable.
## Actual behaviour
When using a `<View>` to wrap `<Icon>` and `<Picker>` in the same row the <Picker> disappears in Android (In IOS it's fine) and the area is not touchable.
## Steps to reproduce (code snippet or screenshot)
```js
<View style={{flexDirection: 'row'}}>
  <Icon name='md-pin'/>
  <Picker iosHeader="Select one"
          mode="dropdown"
          selectedValue={selected}
          style={styles.picker}
          onValueChange={this.onValueChange}>
          {newItems.map((item, i) => {
              return <Item label={item.label} value={item.value} key={item.value}/>
          })}
  </Picker>
</View>
```
## Screenshot of emulator/device
IOS is good:
![image](https://cloud.githubusercontent.com/assets/3540825/22544703/9791fd96-e93e-11e6-874d-87080bcf84a0.png)
Android is not:
![image](https://cloud.githubusercontent.com/assets/3540825/22544721/a28d498a-e93e-11e6-9750-8c6fbb41224b.png)
## Is the bug present in both ios and android or in any one of them?
Present only in Android.
## Any other additional info which would help us debug the issue quicker.
Without using flexDirection: 'row', android is showing the picker correctly.

Another question while we're at it:
How can I add the dropdown arrow that Android has also in IOS?

Thanks in advance!

**Comment by sankhadeeproy007** [comment]
Maybe you can try increasing the width of the containing View and check.
One 'hacky' way to add the dropdown arrow would be to add an Icon and position it absolutely inside the parent `View`

**Comment by BrunoQuaresma** [comment]
Same issue here

**Comment by ilaif** [comment]
@sankhadeeproy007 Thanks! I've tried altering the width of the `<View>` but the picker is still invisible. Any idea how to tackle this issue? I think it's a very basic requirement to be able to position the picker without breaking, thanks again :)

**Comment by mativs** [comment]
I've temporary solved the issue setting {flex:1} on the picker element. I know it's not the ideal solution and haven't try it outside the emulator, but maybe helps.

**Comment by shivrajkumar** [comment]
Use version 2.x

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.4.4](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.4.4)

**Comment by kiranchenna** [comment]
here is my solution for nativebase

import {Row, Col, Icon, Form, Picker} from "native-base"`

<Row style={{ alignItems: "center" }}>
            <Col style={{ flex: 2 }}>
              <Icon name="pin" style={[styles.primaryColor]} />
            </Col>
            <Col style={{ flex: 7 }}>
              <Form>
                <Picker iosHeader="Select one"
          mode="dropdown"
          selectedValue={selected}
          style={styles.picker}
          onValueChange={this.onValueChange}>
          {newItems.map((item, i) => {
              return <Item label={item.label} value={item.value} key={item.value}/>
          })}
  </Picker>
              </Form>
            </Col>
          </Row>`
